pkg/api/types.go

```diff
@@ -388,8 +391,21 @@ const (
 
 	// MountOptionAnnotation defines mount option annotation used in PVs
 	MountOptionAnnotation = "volume.beta.kubernetes.io/mount-options"
+
+	// StorageTopologyConstraint defines topology constraints of a PV
+	// Value is type []TopologyConstraint
+	AlphaStorageTopologyConstraintsAnnotation = "volume.alpha.kubernetes.io/topology-constraints"
 )
 
+// TopologyConstraint contains label information that the resource is constrained to
```

Comment:
Regarding 'optional values'.  I was assuming at least one value must be specified.  What is the behavior if no values are specified?